multimodal/src/autogluon/multimodal/optimization/lit_distiller.py

```diff
@@ -199,6 +236,55 @@ def _compute_soft_label_loss(
         )
         return loss
 
+    def _compute_intermediate_loss(
+        self,
+        student_output: dict,
+        teacher_output: dict,
+    ):
+        student_result = student_output[self.student_model.prefix][FEATURES].squeeze(dim=1)
+        teacher_result = teacher_output[self.teacher_model.prefix][FEATURES].squeeze(dim=1)
+
+        teacher_result = self.intermediate_adaptor(teacher_result)
```

Comment:
Do we also need an adaptor for the student output features? It looks a little bit strange that we only project the teacher output features.

Reply:
I reviewed [FitNet](https://arxiv.org/pdf/1412.6550.pdf) and found that the approach was to upsampling student output to size of teacher's output. I'll change the adaptor and do the experiments again.

Reply:
We can attach the FitNet reference as a code comment.

Reply:
I changed the adaptor from teacher->student to student->teacher. And added the FitNet reference.